### DISCUSSION TITLE
Add support for justETF

### DISCUSSION THREAD

**Discussion by neugartf** [ananthakumaran/paisa#196] [Ideas]
Hey! Awesome application you created here :muscle:.

I was thinking about adding support for justETF, since you can't get many ETFs via yahoo (like `LU0950674332`). 
Unfortunately, I see just a python implementation for this https://github.com/druzsan/justetf-scraping. Any suggestion how to integrate this with go?

Thank you so much :bow: !

**Comment by ananthakumaran** [comment]
It has to be implemented in go, there is no shortcut to it. You can check the existing scraper. But before going there I would suggest looking for scrappers that can generate ledger price db https://ledger-cli.org/doc/ledger3.html#Commodity-price-histories

I still haven't thought much about how the maintenance of the scrappers would work. Whatever supported by paisa as of today will be maintained by me because I am using it personally, but for others it might become complicated. I don't want to have a situation where paisa ends up with a lot of broken scrappers, which is very possible for scrappers with no documented API like justETF
